{"uuid": "a4a5aa42-3f6f-4008-bb2f-596fb940c5fc", "counter": 5, "stdout": "\u001b[0;31mfatal: [192.168.10.5]: FAILED! => {\"changed\": false, \"msg\": \"command timeout triggered, timeout value is 30 secs.\\nSee the timeout setting options in the Network Debug and Troubleshooting Guide.\"}\u001b[0m", "start_line": 4, "end_line": 5, "runner_ident": "77f1d320-9049-4605-9b6c-7493a15a1015", "event": "runner_on_failed", "pid": 470217, "created": "2025-04-15T02:01:08.076469+00:00", "parent_uuid": "000c2948-06b9-c0e8-9bb8-000000000004", "event_data": {"playbook": "ping.yml", "playbook_uuid": "03fd118b-db2b-4787-a47c-ed3d1e49fd35", "play": "Connectivity Test", "play_uuid": "000c2948-06b9-c0e8-9bb8-000000000002", "play_pattern": "all", "task": "Ping a remote device from the UI", "task_uuid": "000c2948-06b9-c0e8-9bb8-000000000004", "task_action": "ios_command", "resolved_action": "cisco.ios.ios_command", "task_args": "", "task_path": "/home/kali/Desktop/fyp/networkproject/networkmanagement/ansible/ping.yml:5", "host": "192.168.10.5", "remote_addr": "192.168.10.5", "res": {"msg": "command timeout triggered, timeout value is 30 secs.\nSee the timeout setting options in the Network Debug and Troubleshooting Guide.", "invocation": {"module_args": {"commands": ["ping "], "match": "all", "retries": 9, "interval": 1, "wait_for": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-04-15T02:00:36.012651+00:00", "end": "2025-04-15T02:01:08.076275+00:00", "duration": 32.063624, "ignore_errors": null, "event_loop": null, "uuid": "a4a5aa42-3f6f-4008-bb2f-596fb940c5fc"}}